debian/control

```diff
@@ -2,13 +2,13 @@ Source: rpiboot
 Section: utils
 Priority: optional
 Maintainer: Serge Schneider <serge@raspberrypi.com>
-Build-Depends: debhelper (>= 9), libusb-1.0-0-dev, pkg-config, rpi-eeprom (>= 22)
+Build-Depends: debhelper (>= 9), libusb-1.0-0-dev, pkg-config, rpi-eeprom (>= 23)
```

Comment:
If you are only installing files from `firmware/`, what is `Build-Depends: rpi-eeprom` for?

Reply:
I guess that can be removed for now since it's getting the files from the submodule. Eventually it should probably get the files from rpi-eeprom